### DISCUSSION TITLE
Are Cosserat Rods Suitable for Non-Shearing Rods?

### DISCUSSION THREAD

**Discussion by Rufnacous** [GazzolaLab/PyElastica#227] [Q&A]
Hello,

Following my previous question, I'd like to ask if PyElastica (and Cosserat Rods in general) is suitable for simulating material that undergoes very little (negligible) shear deformation, instead primarily undergoing bending deformation.

I can reduce the Poisson ratio for my material to close to -1, to make the shear matrix very stiff, but this overall results in very stiff equations. Is there another way I should be approaching this?

Thanks :)

**Comment by armantekinalp** [comment]
In general shear deformations becomes important for low slender ratio (L/D) ratio rods. If you slenderness ratio is greater than 10 generally shear effects are not that important and Cosserat Rod approaches Kirchoff rod. Figure 4 of [this paper](https://ieeexplore.ieee.org/document/7882704)

As you suggested you can also increase the shear components in shear matrix but it will make simulation stiffer and requires smaller time-step.
